etc/config/csharp.defaults.properties

```diff
@@ -0,0 +1,9 @@
+compilers=csharp
```

Comment:
This should be `compilers=&dotnet6` so the whole group is found

Reply:
No problem at all! I've left things I didn't understand, but I'm happy to make changes based on feedback.